config/vufind/BrowZine.ini

```diff
@@ -68,6 +73,26 @@ fullTextFile = "PDF Full Text|browzine-pdf|https://assets.thirdiron.com/images/i
 retractionNoticeUrl = "View Retraction Notice|browzine-retraction"
 expressionOfConcernNoticeUrl = "View Expression of Concern Notice|browzine-concern"
 
+; The bestIntegratorLink service is not yet documented on that wiki page, but it provides
+; "the best link available" according to Third Iron's algorithm. See
+; https://support.thirdiron.com/support/solutions/articles/72000569332-open-access-linking-and-configuration-options).
+; The label and icon values specified here are defaults, but are overridden based on the
+; type of "best" link actually provided for a given title. See [BestIntegratorLinks] below. 
+;bestIntegratorLink = "Get full text|browzine-best"
+
+; When 'bestIntegratorLink' is enabled in DOIServices, this section defines what label and
+; icon to display based on the type of "best" link, overriding defaults in BrowzineFactory.
+; If this section is enabled but an individual link type encountered is not, the label and
+; icon will come from 'bestIntegratorLink' above.
+; See also useBrowzineLabel within [IdentifierLinks] above.
+;[BestIntegratorLinks]
+;fullTextFile = "PDF Full Text|browzine-pdf|https://assets.thirdiron.com/images/integrations/browzine-pdf-download-icon.svg"
+;retractionNoticeUrl = "View Retraction Notice|browzine-retraction"
+;expressionOfConcernNoticeUrl = "View Expression of Concern Notice|browzine-concern"
+;contentLocation = "view_article|browzine-view"
+;documentDeliveryFulfillmentUrl = "Request full text|browzine-delivery"
+;linkResolverOpenUrl = "access_options|browzine-link-resolver"
```

Comment:
This one means "A link to the library’s Link Resolver URL for this article", and the `recommendedLinkText` is "Access Options".  Everyone who works at a library has strong opinions on what the link resolver text should be, and I hate to introduce yet another translation string for the answer, so in theory I could reuse the `Get full text` translation string.

Reply:
Ok, I went with 
> full_text_options = "Check full text options"

I decided I like the word "options" over "availability", because the options can include things like ILL or even purchase requests which go beyond an availability that can be immediately checked.  And I think "check" is probably good but not sure enough to make it part of the key.